Jenkinsfile

```diff
@@ -0,0 +1,60 @@
+job('e2e-tests') {
+    concurrentBuild()
+
+    scm {
+        git {
+            remote {
+                github('coreos/prometheus-operator')
+                refspec('+refs/pull/*:refs/remotes/origin/pr/*')
+            }
+            branch('${sha1}')
+        }
+    }
+
+    wrappers {
+        credentialsBinding {
+            amazonWebServicesCredentialsBinding{
+                accessKeyVariable('AWS_ACCESS_KEY_ID')
+                secretKeyVariable('AWS_SECRET_ACCESS_KEY')
+                credentialsId('Jenkins-Monitoring-AWS-User')
+            }
+            usernamePassword('QUAY_ROBOT_USERNAME', 'QUAY_ROBOT_SECRET', 'quay_robot')
+        }
+    }
+
+    triggers {
+        githubPullRequest {
+           useGitHubHooks()
+            orgWhitelist(['coreos-inc'])
+        }
+    }
+
+    steps {
+        shell('docker build -t cluster-setup-env scripts/jenkins/.')
+    }
+
+    steps {
+        shell('docker run --rm -v /var/jenkins/workspace/e2e-playground:/var/jenkins/workspace/e2e-playground -v /var/run/docker.sock:/var/run/docker.sock cluster-setup-env /bin/bash -c "cd /var/jenkins/workspace/e2e-playground && make crossbuild"')
+    }
+
+    steps {
+        shell('docker build -t quay.io/coreos/prometheus-operator-dev:$BUILD_ID .')
+        shell('docker login -u="$QUAY_ROBOT_USERNAME" -p="$QUAY_ROBOT_SECRET" quay.io')
```

Comment:
@glerchundi @fabxc As we are using the Jenkins credentials plugin it automatically masks them in the log output: `docker login -u=**** -p=**** quay.io`. Thank you for paying attention though! Good feedback.